doc/articles/uno-howto-create-a-repro.md

```diff
@@ -60,6 +60,18 @@ If you're inside of Visual Studio 2022:
 
 Once done, you can zip the folder and send it to github in your issue or discussion.
 
+# [**Windows (Console)**](#tab/windows-console)
+Using the command prompt:
+- Navigate to your sample's root folder
+- Type `bitsadmin /transfer /download "https://raw.githubusercontent.com/github/gitignore/main/VisualStudio.gitignore" "%cd%\.gitignore"`
```

Comment:
This is nice (actually, I just learned this exists), but it isn't easier just to do 

```
dotnet new gitignore
```

I imagine this command doing the same as your command above.